### PR TITLE
Added a note about `sonata_type_model` requisite

### PR DESCRIPTION
**Pull request by lelledaniele**

### CODE REVIEW

File: Resources/doc/reference/form_types.rst [hunk 1]
```diff
@@ -49,6 +49,11 @@ Note that the third parameter to ``FormMapper::add()`` is optional so
 there is no need to pass in an empty array, it is shown here just to demonstrate
 where the options go when you want to use them.
 
+.. tip::
+
+    You need to create `ImageAdmin` class in this case to use `sonata_type_model` type.
```
Comment (OskarStark) [inline Resources/doc/reference/form_types.rst]: please use two `, as this is rst, no markdown:

```diff
- `ImageAdmin`
+ ``ImageAdmin``
```

Comment (lelledaniele) [inline Resources/doc/reference/form_types.rst]: Hi @OskarStark,

sorry, I fixed.

Thanks

File: Resources/doc/reference/form_types.rst [hunk 2]
```diff
@@ -51,7 +51,7 @@ where the options go when you want to use them.
 
 .. tip::
 
-    You need to create `ImageAdmin` class in this case to use `sonata_type_model` type.
+    You need to create ``ImageAdmin`` class in this case to use `sonata_type_model` type.
```
Comment (OskarStark) [inline Resources/doc/reference/form_types.rst]: same for the others ;-)

Comment (lelledaniele) [inline Resources/doc/reference/form_types.rst]: Hi @OskarStark,

done so sorry.
Now should be fine.

Thanks

**Comment by OskarStark** [comment]
@lelledaniele why do you target `master` instead of `3.x` for this?

**Pull request closed without merging**